packages/grid/data-grid/src/tests/layout.DataGrid.test.tsx

```diff
@@ -41,6 +51,7 @@ describe('<DataGrid /> - Layout & Warnings', () => {
     });
 
     it('should resize the width of the columns', async () => {
+      clock.restore();
```

Comment:
Why to restore again?  It's already called in `afterEach`.

Reply:
because this one needs to wait for raf due to autosizer.

Reply:
I moved the clock one scope above.